Update openid-connect-4-ida-claims.md

tmiya0213 opened the pull request.

tmiya0213 committed:
Update openid-connect-4-ida-claims.md

yviscuit commented on openid-connect-4-ida-claims.md:
```diff
@@ -64,23 +64,23 @@ organization="KDDI Corporation"
 ## Additional Claims about End-Users {#userclaims}
 
-This specification defines the following Claims for conveying End-User data in addition to the Claims defined in the OpenID Connect specification [@!OpenID] and the OpenID Connect for Identity Assurance specification [@!OpenID4IDA] and in any other context that a JWT (as per [@RFC7519]) may be used:
+この仕様は, ([@RFC7519]に従う)JWTを利用できるあらゆるコンテキストにおいて, OpenID Connectの仕様[@!OpenID]及びOpenID Connect for Identity Assuranceの仕様[@!OpenID4IDA]で定義されるClaimsに加えて, エンドユーザーのデータを伝達するための次のClaimsを定義している.
```
どこに掛かってるか読み解くの難しいのですが、最後の "and in any other ...." は OIDC/IDA の定義と and なのかなと思ったのですが、どうでしょうか？  
こんな感じです。

> この仕様は、OpenID Connect 仕様 および ...... Claim に加えて、JWT が使われるかもしれないあらゆるコンテキストにおいて、End-User データを伝達するための....

tmiya0213 replied:
修正ありがとうございます！
いただいたコメントを元に修正しました

yviscuit commented on openid-connect-4-ida-claims.md:
```diff
@@ -298,55 +298,55 @@ Claim Name:
 
 Specification Document(s):
-: Section [Claims](#claims) of this document
+: このドキュメントのセクション[Claims](#claims)
```
好みもありそうですが、"このドキュメントの Claims セクション" としたほうが読みやすいかな？と。いかがでしょうか。

tmiya0213 replied:
ありがとうございます！
ご指摘の通りですので、修正しました（他の部分に関しても同様の修正を実施済みです）

yviscuit commented:
https://github.com/openid-foundation-japan/openid-connect-4-identity-assurance/pull/5#issuecomment-1988470775 同様に、元の英文を HTML のコメントアウト形式で残しておいていただけますでしょうか。

tmiya0213 committed:
Update openid-connect-4-ida-claims.md

ご指摘ありがとうございました
作法がわからないため、他のドキュメントと同様に再度プルリクを投げさせていただきます、、

以下、変更点となります
・セクションは前後で入れ替えて対応しました
・ご指摘のあった部分の訳は変更したので、再度ご確認いただければと思います
・原文は全てに追加しました（漏れがあればご指摘ください）

yviscuit commented:
ありがとうございました！
